application/Dockerfile

```diff
@@ -1,13 +1,5 @@
 FROM  openjdk:11.0.7-jdk
 
-ARG TODO_COGNITO_USER_POOL_ID
-ARG TODO_COGNITO_CLIENT_ID
-ARG TODO_COGNITO_CLIENT_SECRET
-
-ENV TODO_COGNITO_USER_POOL_ID=${TODO_COGNITO_USER_POOL_ID}
-ENV TODO_COGNITO_CLIENT_ID=${TODO_COGNITO_CLIENT_ID}
-ENV TODO_COGNITO_CLIENT_SECRET=${TODO_COGNITO_CLIENT_SECRET}
```

Comment:
We should try not to pass env vars like this, and instead inject them via CloudFormation (see `TaskDefinition.ContainerDefinitions.Environment` in `service.yml`).